src/docker-osx-dev

```diff
@@ -226,9 +225,9 @@ function do_rsync {
 #
 function sync {
   local readonly paths_to_sync=("$@")
-
-  for path in "${paths_to_sync[@]}"; do
-    do_rsync "$path"
+  for pathRsync in ${paths_to_sync[@]}
```

Comment:
Huh. I've always seen the `for` loop syntax quote the array, and that's what I use in many other places through out the script. Why would it be different here?